src/routes/(unauthed)/+page.svelte

```diff
@@ -34,6 +35,9 @@
       return;
     }
 
+    await calendars.select(calendar.id);
+    await calendars.subscribe(calendar.id);
```

Comment:
This should go into the middleware API methods. I guess we need a `onboarding.ts` file in `api` which holds the whole "business logic" for onboarding via an invite code, this should then group the resolving logic plus selecting the calendar?